test/test_atomic_features.py

```diff
@@ -16,9 +16,9 @@ def test_atomic_haddock():
         # - comment the first line (E0=1)
```

Comment:
I did not understand test_atomic_haddock() and it's use

Reply:
Is this something I changed? It's not supposed to show up here. I'll look into it.